File: Shrinivas_java_training/session1/controlflow/MultiplicationTable.java
package Shrinivas_java_training.session1.controlflow;

import java.util.Scanner;

public class MultiplicationTable {

    // print multiplication table of a number
    public static void getMultiplication(int num) {
        for (int i = 1; i <= 10; i++) {
            System.out.println(num + " * " + i + " = " + (num * i));
        }
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter a number: ");

        if (!scanner.hasNextInt()) {
            System.out.println("Please enter an integer.");
            scanner.close();
            return;
        }

        int num = scanner.nextInt();

        if (num == 0) {
            System.out.println("Multiplication table of 0 will always be 0");
        }

        getMultiplication(num);

        scanner.close();
    }
}

File: Shrinivas_python_training/python_basic/loops/multiplication_table.py
"""
13. Print multiplication table of a number.
"""


def print_multiplication_table(num: int) -> None:
    """
    This function prints the multiplication table of a given number.
    """

    for i in range(1, 11):
        print(f"{num} x {i} = {num * i}")


# MAIN PROGRAM

number = int(input("Enter a number: "))
print_multiplication_table(number)